draft-ietf-quic-transport.md

```diff
@@ -1131,6 +1131,13 @@ client MUST terminate with a QUIC_VERSION_NEGOTIATION_MISMATCH error code if
 version negotiation occurred but it would have selected a different version
 based on the value of the supported_versions list.
 
+The position and size of the version validation fields in the transport
+parameters MUST NOT change between different versions of QUIC.  If a version of
+QUIC needs to define a new format for transport parameters, it MUST define and
+use a new TLS extension rather than redefine the layout of the existing
```

Comment:
This is a bizarre solution to this problem.  Why not say that it must define a version that indicates a change of format and base new formats on that?  Or something of that sort?

At the point we're doing version negotiation, TLS hasn't come into play, so I don't think this works.